xtask/src/commands/release/apply_plan.rs

```diff
@@ -0,0 +1,200 @@
+use std::{path::Path, process::Command};
+
+use anyhow::{Context, Result, bail, ensure};
+use clap::Args;
+
+use crate::{
+    cargo::CargoToml,
+    commands::{release::plan::Plan, update_package},
+};
+
+#[derive(Debug, Args)]
+pub struct ApplyPlanArgs {
+    /// Only make code changes, do not commit or push.
+    #[arg(long)]
+    dry_run: bool,
```

Comment:
If I run `--dry-run` to make code changes, then without, will it break?

Reply:
I suppose the `ensure_workspace_clean` tells me that's a no :D. Not sure if this is a use case we want to support (we can just reset --hard and go again) but it might be a bit annoying.

Reply:
I'll add a dry-run option to the version bump code so it's a true dry run.